org.eclipse.wildwebdeveloper/src/org/eclipse/wildwebdeveloper/debug/AbstractRunHTMLDebugTab.java

```diff
@@ -148,6 +190,17 @@ public void initializeFrom(ILaunchConfiguration configuration) {
 			this.argumentsText.setText(configuration.getAttribute(AbstractHTMLDebugDelegate.ARGUMENTS, "")); //$NON-NLS-1$
 			this.workingDirectoryText.setText(
 					configuration.getAttribute(AbstractHTMLDebugDelegate.CWD, pathOrEmpty(getSelectedProject())));
+			this.urlText.setText(configuration.getAttribute(ChromeRunDAPDebugDelegate.URL, "")); //$NON-NLS-1$
+			if (urlText.getText().isEmpty()) {
```

Comment:
I ended up just checking if the URL text is empty as opposed to the File parameter so that by default, the File radio button will be selected. 

Also, I wasn't able to do `configuration.getAttribute(ChromeRunDAPDebugDelegate.URL, null)`, it seemed to result in a compile error (or at least, Eclipse didn't like it). However, this works in my testing.